### ISSUE TITLE
Add a FAQ?

### ISSUE THREAD

**Issue by jashapiro** [AlexsLemonade/refinebio-examples#375]
### Background

While adding #374 , I had the thought that maybe a FAQ for our examples might be worth having. 

### Problem

We have some _opinions_ about coding style, etc. that might be somewhat unfamiliar to people. In training, we often get questions about things like:
* "Why do you use `::` so much?"
* "Why do you use `read_tsv()` instead of `read.delim()`?" 
* "What if I want to use python?"

 Maybe laying some of these out explicitly would be something we could do. 
As we test the example pages, more things might come up that might be best addressed by a short FAQ entry than by any major modifications. This would be a nice place to centralize some of these questions.

**Comment by cansavvy** [comment]
I feel like this is similar but not entirely overlapping with the idea of #117.

**Comment by jashapiro** [comment]
> I feel like this is similar but not entirely overlapping with the idea of #117.

Agreed! I forgot about that one. I guess I would call it an expansion of #117?

**Comment by cansavvy** [comment]
Right, so I'm not opposed to having both an FAQ and a Glossary type thing, but as we have these ideas currently, they aren't giving much distinct benefit. 

But neither has been dove into yet. One idea, is it's possible there's information that would more naturally be covered in an FAQ format as opposed to a Glossary. It could be more specific things that show up in code like what you mention here would be addressed in a Glossary but "bigger picture items would better benefit from a FAQ format. 

Not sure at this point.

**Comment by cansavvy** [comment]
If we do both, they could both probably go in a folder called "appendix" or something.
